elide-graphql/src/test/java/graphqlEndpointTestModels/Book.java

```diff
@@ -15,11 +15,12 @@
 import com.yahoo.elide.annotation.OnUpdatePreSecurity;
 import com.yahoo.elide.annotation.ReadPermission;
 import com.yahoo.elide.annotation.UpdatePermission;
-import com.yahoo.elide.graphql.GraphQLEndointTest;
+import com.yahoo.elide.graphql.GraphQLEndpointTest;
 import com.yahoo.elide.security.RequestScope;
 
 import graphqlEndpointTestModels.security.CommitChecks;
 import graphqlEndpointTestModels.security.UserChecks;
+import lombok.Builder;
```

Comment:
Is this Unused?

Reply:
This is used in a javadoc reference.

```
/**
 * Model for book.
 * <p>
 * <b>CAUTION: DO NOT DECORATE IT WITH {@link Builder}, which hides its no-args constructor. This will result in
 * runtime error at places such as {@code entityClass.newInstance();}</b>
 */
```

Reply:
I meant Builder.  I don't see you using the import.